Simplenote/Controllers/WidgetController.swift

```diff
@@ -0,0 +1,37 @@
+import WidgetKit
+
+struct WidgetController {
+    @available(iOS 14.0, *)
+    static func resetWidgetTimelines() {
+        WidgetCenter.shared.getCurrentConfigurations { result in
+            guard case .success(let widgets) = result else {
+                return
+            }
+
+            if widgets.contains(where: { (widget) -> Bool in
+                if widget.configuration as? NoteWidgetIntent != nil {
+                    return true
+                }
+
+                // Add check for ListWidgetIntent when created later
+
+                return false
```

Comment:
Nipticky: 

Since `widget.configuration as? NoteWidgetIntent != nil` already yields a boolean, you can collapse the entire closure as follows:

```
if widgets.contains(where: { widget in
    widget.configuration as? NoteWidgetIntent != nil
})
```